src/extension/regionToSelection.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Selection, TextDocument } from 'vscode'
+import { _Region } from '../shared'
+import '../shared/extension'
+
+export const regionToSelection = (doc: TextDocument, region: _Region | undefined) => {
```

Comment:
>const regionToSelection =  [](start = 7, length = 26)

Can this just be a function?

Reply:
Changed it to a function.

---
In reply to: [435918721](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r435918721) [](ancestors = 435918721)